zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -19,11 +19,11 @@ <h3 class="new_h3">
           Interpreters
         </h3>
         <div class="pull-right" style="margin-top:10px;">
-          <span style="cursor:pointer;margin-right:4px;"
-                ng-click="showRepositoryInfo = !showRepositoryInfo"
-                tooltip-placement="bottom" tooltip="Repository information">
-            <i class="fa fa-cog" ng-style="{color: showRepositoryInfo ? '#3071A9' : 'black' }"></i>
-          </span>
+          <button class="btn btn-default btn-sm"
+                  ng-click="showRepositoryInfo = !showRepositoryInfo">
+            <i class="fa fa-database" ng-style="{color: showRepositoryInfo ? '#3071A9' : 'black' }"></i>
+            Repository Information
```

Comment:
How do you think about changing `Repository Information` to `Repository`? For me, it seems like one button taking too much of space.